ansi_renderer.go

```diff
@@ -48,6 +50,7 @@ type AnsiRenderer struct {
 const (
 	zsh  = "zsh"
 	bash = "bash"
+	pwsh = "pwsh"
```

Comment:
Do you have an idea what this value is in PS5?

Reply:
It looks like it has a shell value of `powershell`. I didn't realize PS5 was one of the supported shells. I'll test things out there and see if I can get this PR to support PS5.

Reply:
I have added Powershell 5 support. The logic worked fine 🎉